aldryn_addons/settings.py

```diff
@@ -49,7 +49,7 @@ def set(self, key, value):
         return IterableUserDict.__setitem__(self, key, value)
 
 
-def load(settings):
+def load(settings, debug=False):
```

Comment:
as mentioned in https://github.com/aldryn/project-template/pull/2 , debug should read from the env var here, as to minimise the footprint that is in all individual website settings.py files (and we don't have control over, after they've been initially created).